### PR TITLE
Update full-stack-modular-development-guide.md

### PR DESCRIPTION
**Pull request by pynchmeister**
## Overview

Changed ```Next.js``` to React.js and ```app``` directory to src

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Review by jcstein — CHANGES_REQUESTED** [review]
hmm, this tutorial is still for a React app using Vite. what's the reason for the change?

**Comment by pynchmeister** [comment]
Yes this tutorial is for a React app using Vite. The inclusion of Next.js seems to be a typo, though...

Screen Shot 2023-01-13 at 2 21 49 PM

File: docs/developers/full-stack-modular-development-guide.md [hunk 1]
```diff
@@ -402,7 +402,7 @@ and secret.
 
 First, create an Infura account and new project for IPFS.
 
-Create a file named `.env.local` in the `app` directory and add the following
+Create a file named `.env.local` in the `src` directory and add the following
```
Comment (jcstein) [inline docs/developers/full-stack-modular-development-guide.md:405]:
```suggestion
Create a file named `.env.local` in the `frontend/` directory and add the following
```

**Review by jcstein — CHANGES_REQUESTED** [review]

**Comment by jcstein** [comment]
read this backwards lol. thank you, lgtm after the requested change. the `.env.local` file will go in the root of the React app directory, so in the `frontend/` dir

**Comment by pynchmeister** [comment]
Sounds good to me, cheers

**Review by jcstein — APPROVED** [review]
LGTM

**Pull request merged**